src/main/java/com/plzgraduate/myongjigraduatebe/user/domain/model/User.java

```diff
@@ -0,0 +1,46 @@
+package com.plzgraduate.myongjigraduatebe.user.domain.model;
+
+
+import lombok.Builder;
+import lombok.Getter;
+
+@Getter
+public class User {
+
+	private String userId;
+
+	private String password;
+
+	private EnglishLevel engLv;
```

Comment:
```suggestion
	private EnglishLevel englishLevel;
```
불필요한 축약 네이밍은 없는게 좋은거 같습니다.

Reply:
[8987d8a](https://github.com/Myongji-Graduate/MyongjiGraduate-BE/pull/154/commits/8987d8ab0b45c69d5c3d475e136324afa3e300c6)
이거 포함해서 롬복 통일 안되있는거하고 도메인 컨벤션 수정!